### PR TITLE
Nested vox

### LINKED ISSUE #1689: Vox lists organization directories
When using organized venvs (that is, venv names with slashes in them), `vox list` will show the organizational directory as a venv.

(Also, it doesn't show envrionments under it.)

```
$ vox create spam/eggs
$ vox list
spam
```

### PR DESCRIPTION
**Pull request by astronouth7303**
Implements full and proper support for "organized" virtual environments in vox.

Basically, you can use slashes in venv names now.

Fixes #1689

### CODE REVIEW

File: xontrib/voxapi.py [hunk 1]
```diff
@@ -162,21 +192,39 @@ def __getitem__(self, name):
...
+            return True
 
     def __iter__(self):
         """List available virtual environments found in $VIRTUALENV_HOME.
         """
-        # FIXME: Handle subdirs--this won't discover eg ``spam/eggs``
-        for x in scandir(self.venvdir):
-            if x.is_dir():
-                yield x.name
+        bin, lib, inc = _subdir_names()
```
Comment (scopatz) [inline xontrib/voxapi.py]: PEP8 don't override builtin names, `bin` here.

**Comment by scopatz** [comment]
Thanks bunches @astronouth7303! Just the one var name issue that I can see.

File: xontrib/voxapi.py (hunk 1 above)
Comment (astronouth7303) [inline xontrib/voxapi.py]: Why didn't my flake8 catch this?

Comment (scopatz) [inline xontrib/voxapi.py]: beats me boss.

**Comment by astronouth7303** [comment]
Fixed.

**Comment by scopatz** [comment]
woohoo!

**Pull request merged**